cookbooks/steinberger/steinberger.prm

```diff
@@ -0,0 +1,219 @@
+# This is a cookbook for a convection model with realistic material properties.
+# Specifically, it uses material properties read in from a look-up table
+# computed with mineral physics software and the projected density approximation.
+# Since it is a quarter of a apherical shell, it also uses periodic boundary
```

Comment:
```suggestion
# Since it is a quarter of a spherical shell, it also uses periodic boundary
```

Reply:
I also don't like the reasoning, one can do many different other boundary conditions (free slip, no slip, etc.)